README.md

```diff
@@ -51,7 +51,57 @@ import { setup } from 'epcis2.js';
 import * as epcis from 'epcis2.js';
 ```
 
-## Building an EPCIS 2.0 JSON document
+## Getting started example
+
+Creating an EPCIS 2.0 document is simple.
```

Comment:
I would formulate this one differently.
It isn't easy to create an EPCIS document, but it is thanks to the SDK ;)